README.md

```diff
@@ -20,6 +20,7 @@
 | [num-traits](https://docs.rs/num-traits/0.2.16)                                    | 0.2.16  |
 | [pyth-sdk](https://docs.rs/pyth-sdk/0.8.0)                                         | 0.8.0   |
 | [pyth-sdk-solana](https://docs.rs/pyth-sdk-solana/0.8.0)                           | 0.8.0   |
+| [seahorse-dev](https://docs.rs/seahorse-dev/0.1.1)                                 | 0.1.1   |
```

Comment:
The crate versions listed here are only the program crates and don't reflect any other crate versions we use so it would better if we deleted this.

```suggestion
```